.github/workflows/ci-cd.yml

```diff
@@ -0,0 +1,68 @@
+name: Brazillian Orders CI/CD Pipeline to EC2 
+
+on:
+  push:
+    branches: [ "orders_api_deployment" ]
+  pull_request:
+    branches: [ "orders_api_deployment" ]
+
+jobs:
+  build_and_push_image:
+    name: Login, Build and Push Docker Image
+    runs-on: ubuntu-latest
+    environment: development
+    steps:
+    - name: Checkout the code
+      uses: actions/checkout@v4
+    - name: Login Dockerhub
+      uses: docker/login-action@v1
+      with:
+          username: ${{ secrets.DOCKER_USERNAME }}
+          password: ${{ secrets.DOCKER_PASSWORD }}
+    - name: remove existing image if it exists
+      run: docker image rm savepeter2/brazillian-orders-api-deployment:latest || true
```

Comment:
Why do you wanna remove this image at all times you build? Conflict sake? If so in the future consider an approach that can avoid this

Reply:
I did this because every time I push code that triggers the workflow, it assigns a new tag named 'latest' to Docker Hub and leaves the existing images untagged, making them unused, which I consider debris that unnecessarily takes up storage space.

Reply:
Or is it cool if I use "docker image prune" everytime I build the image attaching the "latest" tag to the image in order to remove the unused images.